src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV1Spec.groovy

```diff
@@ -669,69 +667,51 @@ Failed to find path with requested bandwidth=$flow.maximumBandwidth/).matches(er
         }
     }
 
-    def "Able to CRUD #flowDescription single switch pinned flow"() {
```

Comment:
Combined two tests into one below (parametrisation for flow type: single sw/different sw)